cmd/pipeline/deploy.go

```diff
@@ -175,14 +176,15 @@ func waitUntilRunning(ctx context.Context, name, namespace string, timeout time.
 }
 
 func getCurrentNamespace(ctx context.Context) (string, error) {
```

Comment:
should this consider the namespace that's coming from the parameter?

Reply:
yeap, this is not call if namespace is defined